fly.toml

```diff
@@ -1,26 +1,31 @@
-# fly.toml file generated
-
 app = "authprod"
 
 kill_signal = "SIGTERM"
 kill_timeout = 5
+processes = []
+
+[deploy]
+  release_command = "/app/bin/migrate"
 
 [env]
+  PHX_HOST = "authprod.fly.dev"
+  PORT = "8080"
 
-[deploy]
-  release_command = "/app/bin/auth eval Auth.Release.migrate"
+[experimental]
+  allowed_public_ports = []
+  auto_rollback = true
 
 [[services]]
-  # http_checks = []
-  internal_port = 4000
-  # processes = ["app"]
+  http_checks = []
+  internal_port = 8080
```

Comment:
is this a deliberate update? 
seems like it should be fine as you're defining the port on `L12` above. 💭

Reply:
Not a deliberate one, this is the file generated by 'fly launch'